exercises/firmware_auditing/README.md

```diff
@@ -0,0 +1,249 @@
+<!--
+Copyright lowRISC Contributors.
+SPDX-License-Identifier: Apache-2.0
+-->
+# Firmware Auditing Exercise
+
+First, make sure to go to the [building the exercises][] section to see how the exercises are built.
+
+[building the exercises]: ../README.md#building-the-exercises
+
+You might find it useful to look at the [auditing firmware][] documentation to get a brief introduction to `cheriot-audit` and the Rego policy language.
+
+[auditing firmware]: ../../doc/auditing-firmware.md
+
+In this exercise, we use the `cheriot-audit` tool to audit the JSON firmware reports produced by the CHERIoT RTOS linker.
+This will let us assert a properties about our firmware images at link-time, guaranteeing desired safety checks.
+This exercise explores a set of self-contained policies which audit a variety of properties, to give an idea of what can be achieved using CHERIoT Audit.
+
+For this exercise, when the `xmake.lua` build file is mentioned, we are referring to [`exercises/firmware_auditing/xmake.lua`][].
+
+[`exercises/firmware_auditing/xmake.lua`]: ../../exercises/firmware_auditing/xmake.lua
+
+## Part 1 - check that firmware contains no sealed capabilities
+
+Policies are written using the *Rego* language, which has syntax that may be unfamiliar.
+If you find yourself confused whilst going through this exercise, it might be helpful to read over the [introduction to Rego][] in  the documentation.
```

Comment:
```suggestion
If you find yourself confused whilst going through this exercise, it might be helpful to read over the [introduction to Rego][] in the documentation.
```